core/src/genesis.rs

```diff
@@ -383,7 +383,6 @@ impl RawGenesisDomainBuilder {
     }
 
     /// Add an account to this domain without a public key.
-    /// Should only be used for testing.
```

Comment:
Why you decide to remove this comment?

Reply:
Because it's not possible to use it anywhere but in tests due to `#[cfg(test)]`